pyproject.toml

```diff
@@ -42,6 +42,7 @@ xml = ["lxml ~= 5.0"]
 
 [project.scripts]
 l10n-build = "moz.l10n.bin.build:cli"
+l10n-build-one = "moz.l10n.bin.build_one:cli"
```

Comment:
Maybe `single` instead of `one`? For some reason, the latter sounds strange to me.